modules/dnn/src/int8layers/pooling_layer.cpp

```diff
@@ -124,6 +125,10 @@ class PoolingLayerInt8Impl CV_FINAL : public PoolingLayerInt8
                 return type == MAX || type == AVE;
             return false;
         }
+        else if (backendId == DNN_BACKEND_INFERENCE_ENGINE_NGRAPH)
+        {
+            return true;
```

Comment:
OpenVINO is optional dependency. Need some availability check here.

Reply:
The checks are mixed across all layers - somewhere `HAVE_INF_ENGINE` checked but somewhere not.